src/main/java/com/zufar/icedlatte/order/entity/OrderItem.java

```diff
@@ -0,0 +1,63 @@
+package com.zufar.icedlatte.order.entity;
+
+import com.zufar.icedlatte.product.entity.ProductInfo;
+import jakarta.persistence.*;
+import lombok.*;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import java.util.UUID;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "order_item")
+public class OrderItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "order_id", nullable = false)
+    private Order order;
+
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
the same here. We can use lazy + entity graphs or projections on demand if needed